packages/ripple-nuxt-tide/lib/templates/axios.js

```diff
@@ -20,7 +20,7 @@ export default function ({ $axios, app, res, isDev }) {
       code = error.response.status
     }
 
-    let responseUrl = error.request?.path || error.request?.responseURL || error.config?.url
+    let responseUrl = error.request?.path || error.request?.responseURL || error.config?.url || ''
```

Comment:
This will fix the error of `TypeError: Cannot read property 'includes' of undefined` which happens when axios get an error without specify the request url.